### DISCUSSION TITLE
How to get involved and start contributing to this project?

### DISCUSSION THREAD

**Discussion by Smurphy000** [delta-io/delta-rs#304] [Q&A] [answered]
Hello,

I have come across this project through references from databricks. I currently use Delta Lake and I like the idea of this project creating bindings through rust. I noticed there is not a section about how to contribute here or what kind of contributions would be helpful at first.

Also, are there any dependencies to do some development on the project? I am currently learning run and I have tried running the tests but have issues with compiling the deltalake-python.

Any information would be helpful, just thought I would reach out through a discussion.

Thanks!

**Comment by houqp** [comment, marked as answer]
Hi @Smurphy000 , welcome to the community. We don't have a strict policy on what type of contributions to accept. As long as the change falls within the scope of the project and is of good quality, we will be happy to merge in. If you are looking for issues to get started, we do have some existing tasks labeled with `help wanted`: https://github.com/delta-io/delta-rs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22.

To develop on the Rust core, we don't require extra dependencies other than the standard rust toolchain. For the python binding, the development environment setup is documented in https://github.com/delta-io/delta-rs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22.

If you run into any error following the documentation, please feel free to file a Github issue so we can get the doc or automation fixed.
